src/samples/Uno.Material.Samples/Uno.Material.Samples.Shared/SamplesPage.xaml.cs

```diff
@@ -52,6 +52,8 @@ private void NavView_Loaded(object sender, RoutedEventArgs e)
 			{ Content = "ToggleSwitch Overview", Icon = new SymbolIcon(Symbol.Next), Tag = "ToggleSwitchSamplePage" });
 			NavView.MenuItems.Add(new NavigationViewItem()
 			{ Content = "FAB Overview", Icon = new SymbolIcon(Symbol.Next), Tag = "FabSamplePage" });
+			NavView.MenuItems.Add(new NavigationViewItem()
+			{ Content = "Cards Overview", Icon = new SymbolIcon(Symbol.Next), Tag = "CardsSamplePage" });
```

Comment:
can you add the menuItem in alphabetical order, to follow with the rest of the items

Reply:
Oh yeah good point, thanks @NicolasChampagne !

Reply:
Done thanks !